posts/forms/compose.py

```diff
@@ -82,10 +82,13 @@ class PostLinkForm(PostForm):
         label="TL;DR",
         required=True,
         max_length=50000,
+        min_length=500,
         widget=forms.Textarea(
             attrs={
+                "minlength": 500,
                 "maxlength": 50000,
                 "class": "markdown-editor-full",
+                "data-listen": "keyup",
```

Comment:
даёт также возможность перечислить другие события, например:
```suggestion
                "data-listen": "keyup focus blur",
```

Reply:
тут опредление дата атрибутов и когда их стоит использовать https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes